CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v1.5.0 (08/04/2020)
+
+**Common**
+
+* `DefaultCache` implementation moved to pimpl.
+* Various warnings fixed.
+* CMake now uses boost repository instead of downloading and unpacking the archive.
+* Method `OlpClientSettingsFactory::CreateDefaultCache` now returns nullptr in case if failed to open a disk cache.
+* Added API to retrieve network statistics. **Not implemented yet.**
+
+**olp-cpp-sdk-authentication**
+
+* Added a new `use_system_time` flag to `olp::authentication::AuthenticationSettings`. You can use it tell authentication module to use system time instead of server time.
```

Comment:
```suggestion
* Added a new `use_system_time` flag to `olp::authentication::AuthenticationSettings`. You can use it to tell the authentication module to work with system time instead of server time.
```